src/main/scala/nl/biopet/utils/ngs/annotation/Feature.scala

```diff
@@ -53,7 +53,7 @@ case class Feature(contig: String,
         case None        => "."
       },
       frame.getOrElse("."),
-      attributes.map(x => x._1 + s""" "${x._2}"""").mkString("; ")
+      attributes.map { case (k, v) => k + s""" "$v"""" }.mkString("; ")
     ).mkString("\t")
```

Comment:
I get that k and v stands for "key" and "value" but can you give more descriptive names here?